tests/com/magento/idea/magento2plugin/inspections/php/GraphQlResolverInspectionTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+package com.magento.idea.magento2plugin.inspections.php;
+
+import com.intellij.testFramework.fixtures.BasePlatformTestCase;
+import java.io.File;
+
+public class GraphQlResolverInspectionTest extends BasePlatformTestCase {
+
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        myFixture.enableInspections(GraphQlResolverInspection.class);
+    }
+
+    @Override
+    protected String getTestDataPath() {
+        return new File("testData/inspections/php/"
```

Comment:
We don’t type paths like that anymore. Please review the abstract classes and create for PHP inspections a similar one. Thank you!